spec/models/applicant_pet_spec.rb

```diff
@@ -5,4 +5,25 @@
     it { should belong_to :pet }
     it { should belong_to :applicant }
   end
+
+  describe 'class methods' do
+    before :each do
+    @shelter = Shelter.create!(name: 'Mystery Building', city: 'Irvine CA', foster_program: false, rank: 9)
+    @applicant = Applicant.create!(first_name: 'John', last_name: 'Dough', street_address: '123 Fake Street', city: 'Denver', state: 'CO', zip: 80205, description: "I'm awesome", status: 'Pending')
+    @pet_1 = @applicant.pets.create!(name: 'Scooby', age: 2, breed: 'Great Dane', adoptable: true, shelter_id: @shelter.id)
+    @pet_2 = @applicant.pets.create!(name: 'Jake', age: 5, breed: 'Pug', adoptable: true, shelter_id: @shelter.id)
+    # require "pry"; binding.pry
+    ApplicantPet.where(applicant_id: @applicant.id).where(pet_id: @pet_1.id).update(approved: true)
+    # ApplicantPet.where(applicant_id: (@applicant.id)).where({pet_id: (@pet_2.id)}).update(approved: true)
+    end
+
+    it "#self.approved?" do
+      # require "pry"; binding.pry
+      expect(ApplicantPet.approved?(@applicant.id, @pet_1.id)).to eq true
+    end
+
+    it "#self.app_status" do
+    expect(ApplicantPet.app_status(@applicant.id, @pet_1.id)).to be_truthy
+    end
```

Comment:
This test does not do much, but I have no clue how to test for an Active record object in an Array. So this is all I could figure out in order to have any test for this.